lms/templates/verify_student/incourse_reverify.underscore

```diff
@@ -0,0 +1,31 @@
+<div id="wrapper-facephoto" class="wrapper-view block-photo face-photo-step">
+  <div class="facephoto view">
+    <h3 class="title"><%- gettext( "Take Your Photo" ) %></h3>
+    <div class="instruction">
+      <p><%= gettext("Course key: ") + courseKey %></p>
+      <p><%= gettext("Checkpoint name: ") + checkpointName %></p>
```

Comment:
Thanks @srpearce.  For now, I'd vote for re-using the existing text (in the verification templates); we're going to need to do a UX/doc review once we complete this feature, so we can revisit the text then if necessary.